.github/workflows/android.yml

```
@@ -14,6 +14,6 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: set up JDK 1.8
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v5
       with:
```

with:
  distribution: 'temurin'
  java-version: '8'

```suggestion
      with:
```